src/blockchain_utilities/blockchain-stats-readme.md

```diff
@@ -0,0 +1,54 @@
+# Monero Blockchain Stats
+
+Monero Blockchain Stats utlity exports daily statistics for the monero blockchain from creation through current state.
+
+## Usage:
+
+See also the utility's help option. `monero-blockchain-stats --help`
+
+From the command line run:
+
+`$ monero-blockchain-stats`
+
+This loads the existing blockchain and prints the results to the terminal. Default printed data includes Blocks per Day, Total Blocks,	Transactions per Day, Total Transactions, Bytes per Day and Total Bytes. The format of the output is in tab delimited csv which is printed to the console. Piping the output of the command to a csv file allows for saving the output of the utilty to a file.
```

Comment:
I'm not a native English speaker, but I don't know if this is right:

> The format of the output is in tab delimited csv which is printed to the console.

"The format (...) is A tab delimited (...)" ?

Also:

> Piping the output of the command to a csv file allows for saving the output of the utilty to a file.

Either I don't understand it fully, or it could be written simplier:
"The output may be stored to a CSV file by piping it."
Or better yet (correct me if I'm wrong) Piping -> Redirecting.
Pipe = `|`
Redirection = `>`

And since we're going that far, that we explain the shell basics, an example command wouldn't hurt.

Reply:
Looks good to me.  
Maybe the `in` in `The format of the output is in tab delimited csv`.

```bash
$ monero-blockchain-stats | save-to-database.sh

$ monero-blockchain-stats > stats-$(date +'%Y-%m-%d').csv
```

Reply:
Hey, this isn't my patch so I will add it to the merge queue as is. Any further suggestions can be done separately.

Reply:
Makes sense.